src/write.rs

```diff
@@ -149,38 +149,74 @@ struct ZipRawValues {
     compressed_size: u64,
     uncompressed_size: u64,
 }
-
+trait Sealed {}
+/// File options Extensions
+#[allow(private_bounds)]
```

Comment:
Please add #[allow(unknown_lints)] as well, since this lint didn't exist in 1.70 (the current MSRV).